README.md

```diff
@@ -8,6 +8,12 @@ use-extension:
   #"@autorest/python": "latest"
   "@autorest/python": "5.0.0-dev.20200211.1"
 
+python: true
+cli: true
+clicommon: true
+require: 
+  - readme.python.md

```

Comment:
doesn't require any $(thisfolder) or something like that in the front?

Reply:
no